### PR TITLE
artifacts: Remove OBS common artifacsts between 2.x and 3.x

### PR DESCRIPTION
**Pull request by gorozco1**
This commit removes artifacts in common between Clear Containers 2.x and
Clear Containers 3.x.

This is to avoid duplicity of code have a consistent repository to build
packages.

The common packages are now moved here: https://github.com/clearcontainers/packaging

Fixes: #1031

Signed-off-by: Geronimo Orozco <geronimo.orozco@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by jodh-intel** [comment]
Ref: https://github.com/clearcontainers/packaging/pull/16.

lgtm

**Comment by jodh-intel** [comment]
@iphutch - could you review the doc changes on this PR please?

File: data/obs-packaging/README.rst [hunk 1]
```diff
@@ -12,23 +12,25 @@ if they are not using `Clear Linux` directly.
...
+Components needed to run Clear Containers 2.x
+=============================================
 
 * `cc-oci-runtime`_: Includes the Clear Containers runtime.
 * `clear-containers-image`_: Includes the mini-OS required to run Clear
   Containers.
 * `clear-containers-selinux`_: Includes the SELinux policy module needed to
   run Clear Containers in environments with SELinux enabled.
-* `linux-container`_: Includes the optimized guest kernel required to run Clear
+* `kernel`_: Includes patches to build an optimized linux kernel required to run Clear
```
Comment (devimc) [inline data/obs-packaging/README.rst]: to run **Intel® Clear Containers** ?

Comment (gorozco1) [inline data/obs-packaging/README.rst]: thanks

**Comment by chavafg** [comment]
qa-passed

**Comment by jcvenegas** [comment]
lgtm

**Comment by gorozco1** [comment]
@iphutch can you take a look at this change please?

**Comment by jcvenegas** [comment]
@rcaballeromx ^

File: data/obs-packaging/README.rst [hunk 2]
```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
...
 When the `cc-oci-runtime`_ or any of its components are updated, we release a
 new package or set of packages.
 
+This repo only contains the `OBS`_ sources to build `cc-oci-runtime`_ and remaining items 
+are under https://github.com/clearcontainers/packaging
+
 Visit our repository at:
 
-- http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/
+- http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.x/
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: Link does not work with the x. The options are: 
http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.0/
http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/
or
http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-staging/

File: data/obs-packaging/README.rst [hunk 3]
```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
 ###############################################
 
 
-The `Clear Containers`_ packages are available via `Open Build Service`_
+The `Intel® Clear Containers`_ packages are available via `Open Build Service`_
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: The addition is not needed, the correct trademark was used already on the title.

File: data/obs-packaging/README.rst [hunk 4]
```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
...
 to allow interested parties the opportunity to try out the technology even
 if they are not using `Clear Linux` directly.
 
 When the `cc-oci-runtime`_ or any of its components are updated, we release a
 new package or set of packages.
 
+This repo only contains the `OBS`_ sources to build `cc-oci-runtime`_ and remaining items
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: This repo only contains the `OBS`_ sources to build the `cc-oci-runtime`_ and the remaining items

File: data/obs-packaging/README.rst [hunk 5]
```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
...
-The `Clear Containers`_ packages are available via `Open Build Service`_
+The `Intel® Clear Containers`_ packages are available via `Open Build Service`_
 to allow interested parties the opportunity to try out the technology even
 if they are not using `Clear Linux` directly.
 
 When the `cc-oci-runtime`_ or any of its components are updated, we release a
 new package or set of packages.
 
+This repo only contains the `OBS`_ sources to build `cc-oci-runtime`_ and remaining items 
+are under https://github.com/clearcontainers/packaging
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: are under: https://github.com/clearcontainers/packaging

File: data/obs-packaging/README.rst [hunk 6]
```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
...
+
 Visit our repository at:
 
-- http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/
+- http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.x/
 
-Included components
-===================
+Components needed to run Intel® Clear Containers 2.x
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: No need to add the Intel®

File: data/obs-packaging/README.rst [hunk 7]
```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
...
-  run Clear Containers in environments with SELinux enabled.
-* `linux-container`_: Includes the optimized guest kernel required to run Clear
+  run Intel® Clear Containers in environments with SELinux enabled.
+* `kernel`_: Includes patches to build an optimized linux kernel required to run Intel® Clear
   Containers
 * `qemu-lite`_: Includes an optimized version of the QEMU hypervisor.
 
-
-.. _`Clear Containers`:  https://clearlinux.org/features/intel%C2%AE-clear-containers
+.. _`Intel® Clear Containers`:  https://clearlinux.org/features/intel%C2%AE-clear-containers
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: Please remove all the Intel® except for the first one.

**Comment by gorozco1** [comment]
@rcaballeromx updated with requested changes. Please review.

**Comment by chavafg** [comment]
qa-passed

File: data/obs-packaging/README.rst [hunk 8]
```diff
@@ -12,22 +12,24 @@ if they are not using `Clear Linux` directly.
...
-* `cc-oci-runtime`_: Includes the Clear Containers runtime.
-* `clear-containers-image`_: Includes the mini-OS required to run Clear
+* `cc-oci-runtime`_: The Clear Containers runtime.
+* `clear-containers-image`_: The mini-OS required to run Clear
   Containers.
-* `clear-containers-selinux`_: Includes the SELinux policy module needed to
+* `clear-containers-selinux`_: The SELinux policy module needed to
   run Clear Containers in environments with SELinux enabled.
-* `linux-container`_: Includes the optimized guest kernel required to run Clear
+* `kernel`_: The patches to build an optimized linux kernel required to run Clear
```
Comment (rcaballeromx) [inline data/obs-packaging/README.rst]: Linux

**Comment by gorozco1** [comment]
@rcaballeromx  done

**Comment by jcvenegas** [comment]
now is failing travis
Failed to run command /usr/bin/git rev-list --no-merges --reverse master..e081982dee95bc190765f72b00c54a454e3737a7: exit status 128 (stdout: , stderr: fatal: Invalid revision range master..e081982dee95bc190765f72b00c54a454e3737a7

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**